Description: Fix conflicting GSocket definition
 If you try to build audacity with a recent release of GLib (I am
 using 2.21.6) it will fail building code that links in the
 wxWidgets GSockets code complaining that GSocket is already defined
 as a typedef by Glib. This patch is a workaround similar to the
 way that the following wxWidgets bug #10883 bug is fixed:
 .
 http://trac.wxwidgets.org/ticket/10883
 .
 Though a better fix might be to split up the code so that the
 code which needs the glib header files and the code that needs
 the wxWidgets header files are separated into different files,
 thus avoiding the issue.
Origin: vendor: http://sourceforge.net/mailarchive/message.php?msg_name=4AA73303.2020409%40sun.com
--- audacity-src-1.3.9/src/AudacityApp.cpp-orig	2009-09-05 03:01:32.893559794 -0500
+++ audacity-src-1.3.9/src/AudacityApp.cpp	2009-09-05 03:06:30.243420380 -0500
@@ -329,7 +329,9 @@ void QuitAudacity()
 ///////////////////////////////////////////////////////////////////////////////
 
 #include <dlfcn.h>
+#define GSocket GlibGSocket
 #include <gtk/gtk.h>
+#undef GSocket
 
 typedef struct _GnomeProgram GnomeProgram;
 typedef struct _GnomeModuleInfo GnomeModuleInfo;
